src/saga/adaptors/pbs/pbsjob.py

```diff
@@ -316,15 +316,13 @@ def _pbscript_generator(url, logger, jd, ppn, gres, pbs_version, is_cray=False,
     if gres:
         pbs_params += "#PBS -l gres=%s\n" % gres
 
-    # escape all double quotes and dollarsigns, otherwise 'echo |'
+    # escape all single quotes, otherwise 'echo |'
     # further down won't work
-    # only escape '$' in args and exe. not in the params
     exec_n_args = workdir_directives + exec_n_args
-    exec_n_args = exec_n_args.replace('$', '\\$')
```

Comment:
losing the '$' in bash script will be a big loss since all bash variable must have '$' to be used.